lib/CLI.js

```diff
@@ -367,16 +419,32 @@ ${packageData.name} v${packageData.version} \
 
   exitWithStatus(error, stats) {
     if (error) {
-      if (error.message) { logger.error(error.message); }
+      if (error.message) {
+        process.stdout.write(
+          `logger: ${JSON.stringify(
+            logger,
+            (key, value) => {
+              // ES6 modules interoperability preserves the module's default export
+              // under the "default" key. When attempted to stringify that,
+              // the process gets into infinite loop due to circular JSON structure.
+              // Explicitly ignoring the "default" key during the conversion to string.
+              return key === 'default' ? null : value;
+            },
+            2
+          )}`
```

Comment:
You might want to include `\n` here, writing to file descriptor doesn't suffix with newline unlike `console.error`.

Reply:
That is no longer needed, as the whole expression has been removed in 7b15907.